sysbuild/CMakeLists.txt

```diff
@@ -340,6 +349,11 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_post_cmake)
     get_property(PM_MCUBOOT_PRIMARY_SIZE TARGET partition_manager PROPERTY PM_MCUBOOT_PRIMARY_SIZE)
     get_property(PM_MCUBOOT_PAD_SIZE TARGET partition_manager PROPERTY PM_MCUBOOT_PAD_SIZE)
 
+    if(SB_CONFIG_MATTER_FACTORY_DATA)
+      get_property(PM_FACTORY_DATA_ADDRESS TARGET partition_manager PROPERTY PM_FACTORY_DATA_ADDRESS)
```

Comment:
What about support for it if there is no Partition Manager support? Due to that our samples for nRF54L15 fail.
For this purpose, we need to read out the `factory_data` partition from a DTS file. We had it in the previous cmake file.

This part seems not to work: 
https://github.com/nrfconnect/sdk-connectedhomeip/pull/438/files#diff-02ef2c92f4124826f9ff5e08958a8644a90f37a4433043d80d5258c38f7341ddR168-R174